clippy.toml

```diff
@@ -2,7 +2,7 @@
 # See full lints list at:
 # https://rust-lang.github.io/rust-clippy/master/index.html
 
-msrv = "1.65.0"
+msrv = "1.70.0"
```

Comment:
I think we can remove it completely, because it defaults to `Cargo.toml`'s `rust-version` at this point.